torch/distributed/_tensor/ops/tensor_ops.py

```diff
@@ -398,6 +399,22 @@ def gather_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> StrategyType:
     return OpStrategy(all_strategies)
 
 
+@register_op_strategy(aten.stack.default, RuntimeSchemaInfo(1, needs_pytree=True))
+def stack_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> StrategyType:
+    args_schema = op_schema.args_schema
+    input_tuple_strategy = args_schema[0]
+    assert isinstance(input_tuple_strategy, TupleStrategy)
+
+    # TODO: If all inputs have the same partial spec, then can keep partial.
```

Comment:
This needs to be made proper. I only unblocked the 1D `clip_grad_norm_` case for now.